docs/sources/reference/glossary.md

```diff
@@ -0,0 +1,194 @@
+page_title: Docker Glossary
+page_description: Glossary of terms used around docker
+page_keywords: glossary, docker, terms, definitions
+
+# Glossary
+
+A list of terms used around the docker project.
+
+## aufs
+
+aufs is a Linux [filesystem](#filesystem) that docker supports as a storage backend.
```

Comment:
Docker (capital D)

Also, should this section include where "aufs" stands for?